### ISSUE TITLE
Focus Trap does not work with iframes

### ISSUE THREAD

**Issue by alexandrzavalii** [Workday/canvas-kit#1097]
## 🐛 Bug Report

focus-trap-js library does not support iframes.
it was designed for maximum performance, it is used in workday grids 2.0, however supporting iframes will make it less performant which is not desirable. 

We can replace it with react-focus-lock instead, which supports iframes
https://www.npmjs.com/package/react-focus-lock

## To Reproduce

Iframe within modal
https://codesandbox.io/s/holy-architecture-qrl1b

## Expected Behavior

Focus should be trapped

**Comment by mathalete** [comment]
Thanks @alexandrzavalii 

Please note that this is an accessibility blocker as it creates a keyboard trap.
